ui/src/app/core/job-manager.service.ts

```diff
@@ -133,4 +140,14 @@ export class JobManagerService {
       .then(response => this.convertToQueryJobsResponse(response.json()))
       .catch((e) => this.handleError(e));
   }
+
+  queryAggregations(timeFrame: TimeFrame, projectId: string): Promise<AggregationResponse> {
+    return this.http.get(`${environment.apiUrl}/aggregations`,
+      new RequestOptions({params: {
+        'projectId':projectId, 'timeFrame':timeFrame
```

Comment:
opt: ES6 supports this syntax for defining objects: `{projectId, timeFrame}` (it uses the variable name as the key)

Reply:
fixed.